docker/drone_env/.env

```diff
@@ -42,7 +42,7 @@ FTP_IP=37.18.77.165
 YANDEX_SHOP_PASS=pass
 EMAIL_HOST_PASSWORD=pass
 # Identify the dependencies folder
-DEPS_DIR=/usr/local/lib/python3.6/site-packages
+DEPS_DIR=/usr/local/lib/python3.7/site-packages
```

Comment:
don't forget plz `docker/env/paths.dist:DEPS_DIR`. And the same file in drone_env too